### DISCUSSION TITLE
Requesting Zoom Phone and Contact Center source connector

### DISCUSSION THREAD

**Discussion by bmercernccer** [airbytehq/airbyte#44107] [New Connector Request]
The existing Zoom connector supports the most common Zoom features, but not Zoom Phone or Zoom Contact Center. 

https://developers.zoom.us/docs/api/rest/zoom-phone-api/
https://developers.zoom.us/docs/api/rest/contact-center-api/

These services generate a lot of data and logs, including call logs, call recordings, queue statistics, user activity, voicemail, etc. 

Zoom Phone and Zoom Contact Center have different levels of functionality but share many common elements. Contact Center adds more complex call center features to the Zoom Phone product. 

My first thought is this would be a single separate connector just to avoid confusing current users of the existing Zoom connector, but it might makes more sense for this to be an enhancement to the existing Zoom connector since they share a common endpoint.

**Comment by mycountrymobile2** [comment]
Requesting the Zoom Phone and Contact Center source connector is essential for enhancing our [Cloud Contact Center](https://www.mycountrymobile.com/) capabilities. By integrating these solutions, we can streamline communication channels, improve customer interactions, and ensure a seamless experience across all touchpoints. The advanced features of Zoom Phone and Contact Center will enable us to better manage our operations, increase efficiency, and deliver superior service in our cloud-based environment. This integration is a strategic move to optimize our overall performance and customer satisfaction.

**Comment by natikgadzhi** [comment]
Hey there! I'm not sure if we have enough live data to play around with them internally, but I'd love to take a look and make one. 

Have you already tried building it in Connector Builder?

**Comment by bmercernccer** [comment]
Not yet, I’m a newbie and have no experience with building my own connections yet.
I’ve been looking at the Zoom APIs, but I’m a sysadmin not a developer, so it’s slow going. Also their documentation seems to be out of date.

[a comment by an automated account is not shown]
